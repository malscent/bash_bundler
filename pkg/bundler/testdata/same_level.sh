#!/bin/bash

source "./other.sh"

echo "Am Not!"