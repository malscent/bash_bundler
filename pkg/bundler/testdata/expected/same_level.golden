#!/bin/bash

################################################################################
#  File:  ./test_scripts/main.sh
#  Bundle Date: 2021-06-09 10:15:57
################################################################################




################################################################################
#  File:  ./test_scripts/other.sh
#  Bundle Date: 2021-06-09 10:15:57
################################################################################


echo "Loser!"


################################################################################
#  End File:  ./test_scripts/other.sh
################################################################################


echo "Am Not!"


################################################################################
#  End File:  ./test_scripts/main.sh
################################################################################


